README.md

```diff
@@ -136,3 +136,31 @@ elif ((current_hour >= 17 && current_hour < 24)) || ((current_hour >= 0 && curre
 fi
 ```
 
+## Main Quest: Moving from a JavaScript file to TypeScript
+
+our first step here is using a package manager(in our case Yarn) to install:
+```
+yarn add typescript
+```
+
+now we're going to rename our .js file to .ts using the following command:
+
+```
+rm program_name.js program_name.ts
```

Comment:
This will actually remove the file 😝 -- the command for renaming (moving) a file is `mv`.

Reply:
I was just testing if you really check the pr's :P ... jk